package.json

```
@@ -137,7 +137,7 @@
     "react-native-linear-gradient": "^2.8.1",
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "1.0.98",
+    "react-native-onyx": "^1.0.100",
```

```suggestion
    "react-native-onyx": "1.0.100",
```

I think we favor a specific version instead of a range like this. Especially with Onyx there can be huge differences. Even though the goal most of the time the code is backwards compatible sometimes the next version does wild stuff :D

---

oh yes thanks, that was a mistake 👍